packages/@aws-cdk/aws-lambda-nodejs/lib/types.ts

```diff
@@ -266,3 +274,29 @@ export enum LogLevel {
   /** Show nothing */
   SILENT = 'silent',
 }
+
+
+/**
+ * SourceMap mode for esbuild
+ * @see https://esbuild.github.io/api/#sourcemap
+ */
+export enum SourceMapMode {
+  /**
+   * Default sourceMap mode - will generate a .js.map file alongside any generated .js file and add a special //# sourceMappingURL=
+   * comment to the bottom of the .js file pointing to the .js.map file
+   */
+  DEFAULT = '',
```

Comment:
Since you are explicitly testing against this, it's better not to use an empty string for an enum value since it evaluates as falsy and could result in trouble.

```suggestion
  DEFAULT = 'default',
```